Extensions/xkit_patches.js

```diff
@@ -236,6 +282,17 @@ XKit.tools.getParameterByName = function(name){
 	}
 };
 
+/**
+ * @return {Object} An overview of the browser's information:
+ *	name<"Google Chrome" | "Mozilla Firefox" | "Apple Safari">: The browser's human-readable name
```

Comment:
is there a standard you're using for these complex type definitions? if so, that's fine, but if not I would prefer a slightly different format